CONTRIBUTING.md

```diff
@@ -4,12 +4,16 @@
 
 - The complete test suite depends on having at least the following installed
   (possibly not a complete list)
-  - git (A sufficiently newer version is required to run pre-push tests)
+  - git (Version 1.8.2 or above is required to run pre-push tests)
```

Comment:
there's also a version needed to run the `pre-merge-commit` tests

Reply:
OK, I changed this to 2.24.0.